src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -0,0 +1,433 @@
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class AppBar : Control
+    {
+        //TODO: This app bar height should be implemented in AppBar style.
+        private float appBarHeight = 72.0f;
+
+        private bool autoNavigationContent = true;
+        private View defaultNavigationContent = null;
+
+        private View appBarNavigation = null;
+        private View appBarTitle = null;
+        private View appBarAction = null;
+
+        /// <summary>
+        /// Creates a new instance of a AppBar.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public AppBar() : base()
+        {
+            defaultNavigationContent = CreateDefaultNavigationContent();
+
+            //Navigation, Title and Action are located horizontally.
+            var linearLayout = new LinearLayout();
+            linearLayout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+            Layout = linearLayout;
+
+            WidthSpecification = LayoutParamPolicies.MatchParent;
+
+            //TODO: This app bar height should be implemented in AppBar style.
+            SizeHeight = appBarHeight;
+
+            if ((AutoNavigationContent == true) && (DefaultNavigationContent != null))
+            {
+                Add(DefaultNavigationContent);
+            }
+        }
+
+        /// <summary>
+        /// Creates a new instance of a AppBar.
+        /// </summary>
+        /// <param name="navigationContent">The content to set to NavigationContent of AppBar.</param>
+        /// <param name="titleContent">The content to set to TitleContent of AppBar.</param>
+        /// <param name="actionContent">The content to set to ActionContent of AppBar.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public AppBar(View navigationContent, View titleContent, View actionContent) : base()
+        {
+            defaultNavigationContent = CreateDefaultNavigationContent();
+
+            //Navigation, Title and Action are located horizontally.
+            var linearLayout = new LinearLayout();
+            linearLayout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+            Layout = linearLayout;
+
+            WidthSpecification = LayoutParamPolicies.MatchParent;
+
+            //TODO: This app bar height should be implemented in AppBar style.
+            SizeHeight = appBarHeight;
+
+            if (navigationContent != null)
+            {
+                NavigationContent = navigationContent;
+            }
+            else if ((AutoNavigationContent == true) && (DefaultNavigationContent != null))
+            {
+                Add(DefaultNavigationContent);
+            }
+
+            if (titleContent != null)
+            {
+                TitleContent = titleContent;
+            }
+
+            if (actionContent != null)
+            {
+                ActionContent = actionContent;
+            }
+        }
+
+		/// <summary>
```

Comment:
just small thing. indentation ^^;

could you check the sites below for easy way of editing code?

- Visual Studio in Windows : https://stackoverflow.com/questions/1435691/visual-studio-format-entire-file

- VS Code in Ubuntu : https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

Reply:
Thank you for the tip! I have fixed the indentation.